platform/jobs/RetirementPipelineJobs.groovy

```diff
@@ -1,4 +1,8 @@
-/* UserRetirementJobs.groovy
+// NOTE: These pipeline-enabled jobs have been retired in favor of the simpler
+// jenkins-native jobs in RetirementJobs.groovy which utilize Parameterized
+// Trigger plugin and a more python centric approach.
```

Comment:
Since these jobs have been retired (temporarily?), I would recommend disabling them directly in the DSL (https://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.jobs.FreeStyleJob.disabled), so they don't get run

Reply:
done.